var/spack/repos/builtin/packages/openfdtd/package.py

```diff
@@ -16,7 +16,9 @@ class Openfdtd(MakefilePackage):
     homepage = "http://www.e-em.co.jp/OpenFDTD/"
     url      = "http://www.e-em.co.jp/OpenFDTD/OpenFDTD.zip"
 
-    version('2.3.0', sha256='10ac70f2ed7160da87dd9222a5a17ca7b72365ee886235359afc48c4fb7b4be4')
+    version('2.6.0', sha256='92f7b92dc55ff6d8fc8c31eda77ca10fe25a5f54b002f2523a3d67f485d77e9f')
+    version('2.3.0', sha256='10ac70f2ed7160da87dd9222a5a17ca7b72365ee886235359afc48c4fb7b4be4',
+            url='http://www.e-em.co.jp/OpenFDTD/old/OpenFDTD_230.zip')
```

Comment:
Is the new version available at a version-specific URL?

Reply:
can not.
The new version download file name does not include version information.